From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Mon, 26 Mar 2012 11:20:32 +0100
Subject: libdir okay

Remove the code that fiddles with ${libdir} from configure.ac, thus
allowing the autotools system in general, and the multiarch code in
particular, to do the right thing, as is its want.
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index bde73ec..f43a5fb 100644
--- a/configure.ac
+++ b/configure.ac
@@ -81,7 +81,7 @@ if test "x$ac_cv_sizeof_void_p" = "x8" ; then
 else
    _lib=lib
 fi
-if test "x$libdir_set_on_cmd" = "xno" ; then
+if false && test "x$libdir_set_on_cmd" = "xno" ; then
    libdir='${exec_prefix}'/"${_lib}"
    AC_SUBST(_lib)
    AC_SUBST(libdir)
